pymc_marketing/mmm/hsgp.py

```diff
@@ -816,6 +865,8 @@ def add_suffix(suffix: str) -> str:
         phi, sqrt_psd = gp.prior_linearized(
             self.X[:, None] - self.X_mid,
         )
+        if self.centered_basis:
+            phi = phi - phi.mean(axis=0).eval()
```

Comment:
How is this related (if any) to the centered parametrization? 

https://github.com/pymc-devs/pymc/blob/main/pymc/gp/hsgp_approx.py#L456-#L470

It seems like a different idea, so shall we change the name (de-mean?)

Reply:
demean_basis?

Reply:
for example . . .

Reply:
Went with `demeaned_basis`. The other parameter is `centered` (past tense) and wanted to mirror that. `drop_first` but that should be removed with the pymc warnings anyway